appveyor.yml

```diff
@@ -18,4 +18,4 @@ test_script:
 - stack setup > nul
 # The ugly echo "" hack is to avoid complaints about 0 being an invalid file
 # descriptor
-- echo "" | stack --no-terminal build --haddock --no-haddock-deps --bench --test :validity-test --no-run-tests
+- echo "" | stack --no-terminal build --haddock --no-haddock-deps --bench --test :validity-test
```

Comment:
Thanks for clarification, my bad. Somehow I didn't think of `:validity-test` as argument of `--test`. I usually run `stack test` and it runs all the tests. Still too lazy to learn more!